include/openssl/crypto.h

```diff
@@ -306,17 +306,20 @@ int CRYPTO_secure_allocated(const void *ptr);
 int CRYPTO_secure_malloc_initialized(void);
 size_t CRYPTO_secure_actual_size(void *ptr);
 size_t CRYPTO_secure_used(void);
+void CRYPTO_get_alloc_counts(int *mcount, int *rcount, int *fcount);
```

Comment:
Uhmmm, with this move, I suspect an additional `make update` is needed, since this got out of the `OPENSSL_NO_CRYPTO_MDEBUG` guard.

Reply:
that was a mistake.  updated commit pushed.